mail-client/alpine/alpine-2.26-r5.ebuild

```
@@ -29,7 +29,7 @@ src_prepare() {
 	default
 
 	# apply patches from upstream git to fix compiler issues
-	local patches=$(find "${WORKDIR}/${P}-patches" -type f -name "0*.patch")
+	local patches=$(find "${WORKDIR}/${P}-patches" -type f -name "0*.patch" | sort)
 	for patch in ${patches}; do
 		eapply "${patch}"
 	done
```

```suggestion
	eapply "${WORKDIR}"/${P}-patches/0*.patch
```
As I already commented in the bug, there is a simpler solution (and maybe simple also means that it is less error-prone).

In any case, if you want to keep the loop then please declare `patch` as a local variable.